crates/bevy_render/src/mesh/mesh/conversions.rs

```diff
@@ -55,6 +58,21 @@ impl fmt::Display for FromVertexAttributeError
             VertexAttributeValues::Float4(..) => "VertexAttributeValues::Float4",
             VertexAttributeValues::Int4(..) => "VertexAttributeValues::Int4",
             VertexAttributeValues::Uint4(..) => "VertexAttributeValues::Uint4",
+            VertexAttributeValues::Short2(..) => "VertexAttributeValues::Short2",
+            VertexAttributeValues::Short2Norm(..) => "VertexAttributeValues::Short2Norm",
+            VertexAttributeValues::Ushort2(..) => "VertexAttributeValues::Ushort2",
+            VertexAttributeValues::Ushort2Norm(..) => "VertexAttributeValues::Ushort2Norm",
+            VertexAttributeValues::Short4(..) => "VertexAttributeValues::Short4",
+            VertexAttributeValues::Short4Norm(..) => "VertexAttributeValues::Short4Norm",
+            VertexAttributeValues::Ushort4(..) => "VertexAttributeValues::Ushort4",
+            VertexAttributeValues::Ushort4Norm(..) => "VertexAttributeValues::Ushort4Norm",
+            VertexAttributeValues::Char2(..) => "VertexAttributeValues::Char2",
+            VertexAttributeValues::Char2Norm(..) => "VertexAttributeValues::Char2Norm",
+            VertexAttributeValues::Uchar2(..) => "VertexAttributeValues::Uchar2",
+            VertexAttributeValues::Uchar2Norm(..) => "VertexAttributeValues::Uchar2Norm",
+            VertexAttributeValues::Char4(..) => "VertexAttributeValues::Char4",
+            VertexAttributeValues::Char4Norm(..) => "VertexAttributeValues::Char4Norm",
+            VertexAttributeValues::Uchar4(..) => "VertexAttributeValues::Uchar4",
             VertexAttributeValues::Uchar4Norm(..) => "VertexAttributeValues::Uchar4Norm",
```

Comment:
I have not added a `_` placeholder so that whoever adds new formats will take care of conversions